tests/features/sitemap.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Check the Sitemap
+  In order to check if the sitemap is available and correct.
+  As an administrator
+  I want to check the sitemap generated is available and correct.
```

Comment:
These are not business goals. A sitemap is not there in order to be able to check if it's available and correct.